### DISCUSSION TITLE
Saving to database

### DISCUSSION THREAD

**Discussion by RizaHKhan** [saulpw/visidata#2055] [Q&A]
I'm opening a database via this command:

`vd -N postgresql://postgres:password@localhost:5433/my-wonderful-database`

However, when I try to edit a cell it will not save. Am I doing something wrong or is saving these changes not even allowed from Visidata?

**Comment by saulpw** [comment]
Hi @RizaHKhan, I believe saving with `z Ctrl+S` should commit the changes.  If it's not working, you could open an issue and provide more information about what you're specifically trying and what you're seeing, so that we might be able to figure out who's doing what wrong.
